pkg/server/server.go

```diff
@@ -480,6 +511,11 @@ func (s *Server) Check(ctx context.Context, req *openfgav1.CheckRequest) (*openf
 	}
 
 	span.SetAttributes(attribute.KeyValue{Key: "allowed", Value: attribute.BoolValue(res.GetAllowed())})
+	requestDurationByQueryHistogram.WithLabelValues(
+		"Check",
```

Comment:
Can we keep the server package and service prefix for this so that it's a FQN for the RPC? That's more standard practice and leaves us well set for introducing a v2 potentially in the future.

Just like we do above in https://github.com/openfga/openfga/pull/950/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R505-R506

Reply:
> Can we keep the server package and service prefix for this so that it's a FQN for the RPC? That's more standard practice and leaves us well set for introducing a v2 potentially in the future.
> 
> Just like we do above in https://github.com/openfga/openfga/pull/950/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R505-R506

I am not sure what you mean.  Currently the [datastoreQueryCount](https://github.com/openfga/openfga/blob/37f48d1b54aa1a69d7a16aa1d0deb4549aef4280/pkg/server/server.go#L473) uses "check" as the method name.  We can change this from "Check" to "check".  However, if we are going to use the FQN, we need to change datastoreQueryCount as well.

Reply:
See https://github.com/openfga/openfga/pull/924 for datastoreQueryCountHistogram grpc_method

Reply:
Right, but we also use `grpc_service` in the other metric. So each observation is fully qualified by the grpc service and grpc method that the metric measures.

I vote we be consistent with what we have currently.

Reply:
So in this case we have 3 dimension then

Reply:
It's consistent with what we're doing elsewhere, but if we want to flatten it into a single dimension instead of two that's where I suggest we `fmt.Sprintf("%s/%s", openfgav1.OpenFGAService_ServiceDesc.ServiceName, "Check")` for example